scripts/globals/additional_effects.lua

```diff
@@ -21,6 +21,24 @@ require("scripts/globals/events/harvest_festivals")
 xi = xi or {}
 xi.additionalEffect = xi.additionalEffect or {}
 
+xi.additionalEffect.procType =
```

Comment:
prolly gonna adopt this change upstream later but with a name change or extra `.layer` of table depth because of the need to eventually support a procType for spikes as well (you may have noticed we don't do spike effect items correctly either, with a bunch of hard coding at the core)